staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -111,6 +113,7 @@ type MutatingWebhook struct {
 	clientManager    config.ClientManager
 	convertor        versioned.Convertor
 	jsonSerializer   runtime.Serializer
+	configclient     admissionregistrationv1beta1.AdmissionregistrationV1beta1Interface
```

Comment:
no need this here, you can just init it using `PostStartHookContext.LoopbackClientConfig` below.

Reply:
The webhook could be running in another apiserver. The admissionregistration API is hosted in the kube-apiserver, so loopback client doesn't work.

Reply:
make sense.